Simplified/en.lproj/Localizable.strings

```diff
@@ -213,6 +211,10 @@
 "LocationRequiredMessage" = "Location information is required for Sign Up. Please allow access to location information in the iOS settings";
 "Error parsing user profile document." = "Error parsing user profile document. Please try again.";
 "Trouble locating DRM in profile document." = "Trouble locating DRM in profile document. Please try again.";
+"SigningInAgree" = "By signing in, you agree to the End User License Agreement.";
+"Want a card for your child?" = "Want a card for your child?";
+"We are unable to read your library card, which is necessary in order to create a dependent card." = "We are unable to read your library card, which is necessary in order to create a dependent card.";
+"Try to sign out, sign back in, then try again." = "Try to sign out, sign back in, then try again.";
```

Comment:
I take it all these strings were run by or came from @killerpuppytails ?

Reply:
not yet, but I am compiling a list of strings for these edge error cases and I was thinking to send it over closer to the end.